tasks/libs/common/utils.py

```diff
@@ -700,3 +700,13 @@ def is_pr_context(branch, pr_id, test_name):
         print(f"PR not found, skipping check for {test_name}.")
         return False
     return True
+
+
+@contextmanager
+def collapsed_section(section_name):
+    section_id = section_name.replace(" ", "_")
+    try:
+        print(f"\033[0Ksection_start:{int(time.time())}:{section_id}[collapsed=true]\r\033[0K{section_name + '...'}")
+        yield section_id
```

Comment:
I don't mind yielding it, but what could the user do with this id?

Reply:
Indeed it was just to follow the `context manager` pattern, this is probably useless